azure-pipelines/templates/steps/credscan-policheck.yml

```diff
@@ -1,27 +1,33 @@
 # File: credscan-policheck.yml
```

Comment:
>credscan-policheck [](http://example.com/codeflow?start=8&length=18)

maybe we can rename this as "security-compliance-check.yml"

Reply:
change applied here 983cd5ca9f5381e5fcba8a6cb3ff3626253c293c